CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java

```diff
@@ -603,6 +603,12 @@ private void showPriorityDialog() {
         alert.getDialogPane().setContent(rulesScrollPane);
         alert.setResizable(true);
 
+        final List<Screen> screens = Screen.getScreensForRectangle(this.getScene().getWindow().getX(), this.getScene().getWindow().getY(),
+                this.getScene().getWindow().widthProperty().get(), this.getScene().getWindow().heightProperty().get());
+
+        alert.setX((screens.get(0).getVisualBounds().getMinX() + screens.get(0).getVisualBounds().getWidth() / 2) - 350);
+        alert.setY((screens.get(0).getVisualBounds().getMinY() + screens.get(0).getVisualBounds().getHeight() / 2) - 170);
```

Comment:
Can you change the magic numbers (350 & 170) to instead use a calculation equal to half the width and height of the `rulesScrollPane` object as appropriate.

Please also fix a similar issue on lines 703 & 704 where a fixed number is being used when it should be calculated as width/2 and height/2 of the `sp` object.